testdata/p4_16_samples/arith-inline-skeleton-d.p4

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
I am not sure what the copyright message should be. Also this file does not include psa.p4, not required?